### PR TITLE
Metadata pane

### PR DESCRIPTION
**Pull request by kirkkwang**
ref: https://github.com/scientist-softserv/iiif_print/issues/154, https://github.com/scientist-softserv/iiif_print/issues/173

essi-metadata-pane

[Allow IIIF Print to handle manifests](https://github.com/scientist-softserv/essi/commit/bd2b8176accf1d4b2829df81c6237a5f7ab8ff82) 

Originally there were a lot of extensions that handled manifest creation
and sanitization.  This commit will hand off those responsibilites to
the IIIF Print gem.

---

[Update IIIF Print](https://github.com/scientist-softserv/essi/commit/94de30ccd953c34136ea6a021eb1dfc800165ab9)

---

[Fix File Manager button](https://github.com/scientist-softserv/essi/commit/018d34e8429cd6f7ff91ea0a8fb962a37a79dd06) 

After installing IIIF Print, the File Manager button throws an error
when navigating from the parent work because the `node` variable is a
`WorkShowPresenter` instead of a `FileSetPresenter`.  To solve this we
find it's `FileSetPresenter` and send that down the line.

### CODE REVIEW

**Review by jeremyf — APPROVED** [review]

File: app/views/hyrax/base/_file_manager_attributes.html.erb [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<%# When navigating to the File Manager from a parent work, the `node`  %>
+<%# is a `WorkShowPresenter` instead of a `FileSetPresenter` which does %>
+<%# not have #extracted_text? which will cause a problem further down.  %>
+<%# To avoid an exception, we use the parent work's `FileSetPresenter`. %>
+<% if node.is_a?(Hyrax::WorkShowPresenter) %>
+  <% node = node.file_set_presenters.find { |fsp| node.file_set_ids.include?(fsp.id) } %>
+<% end %>
```
Comment (jeremyf) [inline app/views/hyrax/base/_file_manager_attributes.html.erb:7]: Non-blocker, but wondering what would happen if you added `#extracted_text?` method to the `Hyrax::WorkShowPresenter`?  Yes, you'd probably need to make a `#file_set_presenter` method.

Comment (kirkkwang) [reply app/views/hyrax/base/_file_manager_attributes.html.erb:7]: That's a good question, the OCR lives on the file sets and not the work itself, so I'm wondering if we would still need to find it's file set anyways? 🤔

**Comment by ShanaLMoore** [comment]
I'm going to merge this since Kirk is out sick today.

**Comment by ShanaLMoore** [comment]
@kirkkwang I must be missing it, but how were you able to resolve the render_as issue? I see that it's working (facetable/clickable title in the metadata pane)

**Comment by kirkkwang** [comment]
@ShanaLMoore 
Yes i got it working here:
https://github.com/scientist-softserv/iiif_print/pull/175/files#diff-fc39c95bd4a1e0a5ca6c4c4ff5dbb0d3d5eec15dd19caa0ffab8a9580452955cR137-R170

I was able to pull out `:indexing` as well in the query.

**Pull request merged**